cmd/starter/main.go

```diff
@@ -74,11 +75,18 @@ func waitForBuild(conn *beanstalk.Conn) {
 			panic(err)
 		}
 
-		repo := string(body)
+		job, err := buildjob.FromBytes(body)
+		if err != nil {
+			// there was a job we could not parse throw it away and continue
+			_ = conn.Delete(id)
+			continue
```

Comment:
not sure how `conn.Delete(id)` works, but important i think to log this error or give it as feedback to the user. i.e. maybe failing silently is not desired?